### DISCUSSION TITLE
Quay.io

### DISCUSSION THREAD

**Discussion by daniellutz** [beaker-project/beaker#136] [General]
Hi @StykMartin ,

I was changing some projects and I was thinking of using Quay.io to store some Docker images, but I have seen that there is already a Beaker organization.

Is there any plan to using [https://quay.io/organization/beaker](https://quay.io/organization/beaker) ?

Thank you

**Comment by StykMartin** [comment]
Yep. We will use it for upstream images.

**Reply by daniellutz** [reply]
We are planning to host some images, i.e. AMB, on quay.io.

Would you prefer of us creating another org and keeping separate downstream/upstream or can we use the beaker org as well?

Thank you!

**Reply by daniellutz** [reply]
We would like to keep some things related to upstream, whenever possible.

**Reply by StykMartin** [reply]
I don't really seen connection here. Stuff like AMB has nothing to do with Beaker in upstream. We can definitely revisit this later on if there will be interesting subject
